doc/classes/String.xml

```diff
@@ -5,6 +5,12 @@
 	</brief_description>
 	<description>
 		This is the built-in string class (and the one used by GDScript). It supports Unicode and provides all necessary means for string handling. Strings are reference counted and use a copy-on-write approach, so passing them around is cheap in resources.
+		To know whether a string contains a substring, use the [code]in[/code] operator as follows:
```

Comment:
For such a massive class like String, I'm not sure if it's best to have this kind of details in the description, when tons of other "must-know" about Strings aren't there. Maybe it would be better to document the `in` operator in the `find` method directly, as an alternative to `find`?

The same would then apply to other classes when they have a `has` or `find` method that matches what `in` does.

It's not bad per se as done here so we could merge it as is, but it seems a bit out of place to me..

Reply:
I changed it so the `in` operator is described in the `find` method (or equivalent) now.